hasura.planx.uk/migrations/1715929881942_run_sql_migration/up.sql

```diff
@@ -0,0 +1,40 @@
+DROP VIEW "public"."feedback_summary";
+
+CREATE OR REPLACE VIEW "public"."feedback_summary" AS
+SELECT 
+    fb.id AS feedback_id,
+    t.slug AS team_slug,
```

Comment:
Apologies for the churn in this view recently, I've renamed `team` to the more explicit and correct `team.slug` so that the relationship can just be called `team` in Hasura.